Makefile

```diff
@@ -18,10 +18,10 @@ push:
 .PHONY: push
 
 run:
-	$(DOCKER) container run --rm \
+	$(DOCKER) container run --rm --platform linux/amd64 \
```

Comment:
Shouldn't you use DOCKER_BUILD_FLAGS  variable from above?

Reply:
good point.  updating